xc/xc7/toolchain_wrappers/vpr_common

```diff
@@ -76,6 +80,7 @@ function parse_args {
      fi
 
      export DEVICE=$DEVICE
+     export BITSTREAM_DEVICE=$BITSTREAM_DEVICE
```

Comment:
Can you explain why we have a `DEVICE` and a `BITSTREAM_DEVICE`?  Seems redundant?

Reply:
Yeah, I should definetely rename these, in the meantime:

- DEVICE = actual name of the device from symbiflow-arch-defs, e.g. `xc7a50t_test`, from which the RR graph and all other files are named after.
- BITSTREAM_DEVICE = xc7 family, e.g. artix7, zynq7. This is used to access the db.

Reply:
But isn't the information redundant?  Doesn't `xc7a50t_test` imply `artix7`?

Reply:
Well, sure, we could get the place_constraints script (or the toolchain wrappers) to infer that `xc7a50t_test` corresponds to the `artix7` family, I didn't really want to rely on implications and have the artix7 explicitly added as an argument for safety.

Reply:
I guess what really doesn't make sense is why a prjxray specific variable is needed in the script called `vpr_common`?  I thought we were trying to create a unified set of scripts, and this seems like it is moving in the wrong direction?

Reply:
So, for context, the problem here was that the place constraints for the IOPADs for the GTP need to respect the way we are building the GTP tiles, and these must be in sync, otherwise constrained sub_tiles won't match, causing routing issues.

It might be possible to store the information on how to build and on how to constrain the IOPADs in the vpr_grid.csv or some additional data file, so we do not need to access prjxray

Reply:
But nothing you just wrote has to do with VPR itself?  `vpr_common` is for handling invocation of VPR.  Handling placement constraints happens before invoke VPR, yes?

Reply:
Well, as part of the toolchain wrappers, the `symbiflow_generate_constraints` part is invoked within the `symbiflow_place` wrapper, therefore the `bitstream_device` argument needed to be passed to the `symbiflow_place` command.

The generate_constraints step consists in calling the constraint generation scripts, prior to invoking the VPR placer.

 In reality, VPR  itself does not need this information at all.

Reply:
I believe that `symbiflow_generate_constraints` should deal with mapping `DEVICE` to `BITSTREAM_DEVICE`, and that we should not be adding more arguments to the top level wrapper. This detail appears to be prjxray specific, and that probably shouldn't leak into the broader scripts.

Reply:
All right, I have added the mapping into the constraints generation script